hydra-node/src/Hydra/Chain/Direct/State.hs

```diff
@@ -324,7 +324,7 @@ close confirmedSnapshot pointInTime OnChainHeadState{ownVerificationKey, stateMa
     ConfirmedSnapshot{snapshot = Snapshot{number, utxo}, signatures} ->
       CloseWithConfirmedSnapshot
         { snapshotNumber = number
-        , closeUtxoHash = hashTxOuts $ toList utxo
+        , closeUtxoHash = hashUTxO @Tx utxo
```

Comment:
Wondering if these are indeed semantically equivalent?

Reply:
They should be. I will change back the definition of `hashUTxO` to be not sorting besides using `toList`.